Adds Detailed Quick Start Instructions (#1)
alexfornuto/romm-wiki: File-Structure.md

> Here is the recommended Folder/File structure in order to ensure your roms are properly identified.  Failure to follow one of these formats **may** work, but is not guaranteed nor supported.

Reviewer: This would be a good place to document or link to documentation describing how to associate different folder names with consoles.

> *library* refers to RomM's internal container path, just replace it in your example with the directory containing your roms.

Reviewer: ```suggestion
*library* refers to RomM's internal container path. Replace it in your example with the directory containing your roms.
```

I may be applying more business documentation logic than is required here, but terms like "just" and "simply" can make processes sound like they should be trivial to the audience, and assuming their knowledge/experience can spell trouble.

#### Recommended Format

> This is the ideal structure to follow, every new installation should follow this unless absolutely necessary.

Reviewer: Regarding "unless absolutely necessary", is is really all that big a deal? AFAIK RomM can be pretty good about associating other directory names nowadays.

Author: It can be really finicky about whether or not it will detect the roms if not in one of the supported structures, unless you modify the config,yml.   For this reason, I used this wording to highly discourage custom structures that would cause support headaches.  Maybe arcaneasada might have some input. and can correct this.

> Inside of the library should be a single folder called ***roms***.  Herein you will place directories containing the platforms' folder name - see [here](https://github.com/rommapp/romm/wiki/Supported-Platforms) for the list of standard supported platforms, and refer [here](https://github.com/rommapp/romm/wiki/Custom-Platforms) for platforms that are not on the list.

Reviewer: ```suggestion
Inside of the library should be a single folder called ***roms***.  Here you should place directories containing the platforms' folder name - see [Supported Platforms](https://github.com/rommapp/romm/wiki/Supported-Platforms) for the list of standard supported platforms, and refer to [Custom Platforms](https://github.com/rommapp/romm/wiki/Custom-Platforms) for platforms that are not on the list.
```

Change summary:

- Simpler words help future translations.
- Softer verb usage.
- Use page titles in link text. "Here" doesn't let the user know where they are being pointed to until they click it, so they can't decide if they want/need to first.

##### Single File ROMs

```/library/roms/platform/rom_files```

![](assets\single-file-structure.svg)

##### Multi-File ROMs

```/library/roms/platform/rom_name/rom_files```

> ![](assets\multi-file-structure.svg)

Reviewer: Here and throughout, having image descriptions benefits the visually impaired.
